react_demo/cpp/index.html

```diff
@@ -0,0 +1,250 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>FROM_C_PLUS_PLUS</title>
```

Comment:
Could you elaborate pls?

On Saturday, June 6, 2015, John Babak <notifications@github.com> wrote:

> In react_demo/cpp/index.html
> <https://github.com/KnowSheet/SimpleDashboard/pull/7#discussion_r31874049>
> :
>
> > @@ -0,0 +1,250 @@
> > +<!DOCTYPE html>
> > +<html>
> > +  <head>
> > +    <meta charset="utf-8">
> > +    <meta http-equiv="x-ua-compatible" content="ie=edge">
> > +    <title>FROM_C_PLUS_PLUS</title>
>
> Can't see PAGE_TITLE_FROM_C_PLUS_PLUS in the referenced commit.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/KnowSheet/SimpleDashboard/pull/7/files#r31874049>.
>


-- 
And, as always, have a nice day!
Dima

http://dimakorolev.com/ | http://www.linkedin.com/in/dimakorolev |
dmitry.korolev@gmail.com | +1 (312) 593 2783